codecov.yml

```diff
@@ -0,0 +1,8 @@
+---
+coverage:
+  notify:
+    slack:
+      default:
+        url: secret:5WK4lHFGNbrytMXL36g63my9P4W6kLCI3sP/vpKELSubtWw0aU4u4wRAa1yb6co5zFhJicC3t4dxZvPhQ63ZXIVVUmqkykam4wjfMguN+GDo7IxUMygmyWzxdzCwLZBRomOS1F2UCvDS29zBfNiF7LudEyUMPuKtpo5lj4jv5tU=
+
+comment: false
```

Comment:
I will reenable the comments. If we decide we don't like it, we can turn it back off.